happy.cabal

```diff
@@ -161,12 +161,14 @@ executable happy
   build-depends: base < 5,
                  array,
                  containers >= 0.4.2,
+                 dom-lt >= 0.2.2,
```

Comment:
It is some kind of a deal

- https://matrix.hackage.haskell.org/#/package/happy
- https://matrix.hackage.haskell.org/#/package/dom-lt

Currently `happy` is buildable with virtually any GHC you can get your hand on. With `dom-lt` it won't be true anymore.

Reply:
Hmm. I wonder if we could remove the lower bound. Then we'd be fine, I think. AFAICT, the lower bound is because of https://github.com/AndreasPK/dom-lt/issues/2, but if 0.2.0 and 0.2.1 were marked broken, the solver wouldn't pick these.

Reply:
That issue says that `dom-lt-0.1.3` is broken as well, so that won't work.

It might be easier to say that `happy-next` would work only with GHC-8.0+. I think it is fine, not small issue, but not huge either.

Reply:
Why exactly is `dom-lt-0.2.2` not compatible with GHC-7? Maybe this can be dealt with in `dom-lt`?

Reply:
> Why exactly is dom-lt-0.2.2 not compatible with GHC-7

Maybe the code would work. I honestly just haven't tried it with GHC 7 as I haven't used that old an GHC in years by now.

If someone put's the effort in to make it work/test it with GHC 7 I have no issue lowering the bounds.